### ISSUE TITLE
Multi-output Xgboost show wrong results.

### ISSUE THREAD

**Issue by lk1983823** [intel/scikit-learn-intelex#1439]
I am training a xgboost model with multiple inputs and outputs. The output feature_dim of the model is 11. Then I use d4p to accelerate the trained model to improve inference performance. Here is my code below:
```
daal_model = d4p.get_gbt_model_from_xgboost(xgbmodel.get_booster())
daal_prediction = d4p.gbt_regression_prediction().compute(test_x, daal_model)
logits = daal_prediction.prediction
```
But when I print the logits, I find the its output shape is (272303, 1). The outputs has only 1 feature_num, and the result is also wrong. 
Thanks for the help. 

**Environment:**
 - OS: [e.g. Ubuntu 18.04]
 - Compiler: [e.g. GCC9.2]
 - Version: [e.g. 2019 Update 4]

**Comment by napetrov** [comment]
@lk1983823  thanks for report. 

In short - multioutput is not supported

Will hold this issue opened for tracking documentation update and model builders update to warn on not supported multioutput models

**Comment by david-cortes-intel** [comment]
Under the new interface for serving GBT models, this now throws an explicit error instead of producing incorrect results.
